.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 helm/memcached-operator/.idea/*
 ansible/memcached-operator/.idea/*
+go/memcached-operator/.idea/*
```

Comment:
You should be able to replace all of these `.idea` lines with `/*.idea/*`.

Reply:
it does not work.

Reply:
done